File: parser/data/atom_test_data/py/py_atom_test_39.py
def atomTest():
    s = [1,2,3,4,5]
    a = 0
    for i in range(0, len(s)):
        if i+1 >= len(s):
            a = -1
        if s[i] < s[i+1]:
            a = s[i]
        else:
            a = -1
    return a

File: evaluation/javascripthon_atom_test_result/py_atom_test_39.js
function atomTest() {
    var a, s;
    s = [1, 2, 3, 4, 5];
    a = 0;
    for (var i = 0, _pj_a = s.length; (i < _pj_a); i += 1) {
        if (((i + 1) >= s.length)) {
            a = (- 1);
        }
        if ((s[i] < s[(i + 1)])) {
            a = s[i];
        } else {
            a = (- 1);
        }
    }
    return a;
}

